AppCreationScripts/apps.json

```diff
@@ -12,7 +12,7 @@
         "replyUrlsWithType": [
           {
             "url": "http://localhost:4200/",
-            "type": "Web"
+            "type": "Spa"
```

Comment:
This is unrelated to the other changes, and would prefer it be made separately when we are ready to update Angular to the auth code flow.

Reply:
I see. I'll revert this back.